style.css

```diff
@@ -547,3 +547,494 @@ hr {
+  }
+
+  .status-bar {
+    display: none;
+  }
+
+  .topnav {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+    width: 1349px;
+    background-color: rgb(60, 58, 57);
+    height: 72px;
+  }
+
+  h1 {
+    color: hsl(25, 100%, 50%);
+    font-family: "Crete Round", sans-serif;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .supporting-text {
+    font-family: "inter", sans-serif;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 32px;
+    color: hsl(40, 100%, 94%);
+    margin-bottom: 12px;
+  }
+
+  .description {
+    display: block;
+    width: 566px;
+    font-family: "inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: rgba(244, 245, 247, 1);
+  }
+
+  .headline {
+    display: flex;
+    background: rgba(28, 26, 1) url(./images/Illustration-header.png) no-repeat;
+    background-position: right;
+    height: 798px;
+    padding: 0;
+    width: 1349px;
+    margin: auto;
+  }
+
+  .hero {
+    padding: 143px;
+  }
+
+  .social-icons {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    left: 40px;
+    bottom: 187px;
+  }
+
+  .social-icons li {
+    margin: 0;
+  }
+
+  /* work-section */
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 120px 0 0 143px;
+  }
+
+  .work-header {
+    width: 315px;
+    font-size: 40px;
+    font-weight: 400;
+    font-family: "Crete Round", serif;
+    line-height: 52px;
+    padding: 20px 0;
+  }
+
+  .btn-card {
+    cursor: pointer;
+    display: inline-block;
+    border: 1px solid;
+    padding: 6px 12px;
+    background: white;
+    color: black;
+    font-size: 15px;
+    line-height: 24px;
+    text-decoration: none;
+    margin-right: 10px;
+  }
+
+  .btn-car {
+    cursor: pointer;
+    display: inline-block;
+    border: 0;
+    padding: 10px 10px;
+    background: rgb(255, 255, 255, 0.24);
+    color: white;
+    font-size: 15px;
+    line-height: 24px;
+    text-decoration: none;
+    margin-right: 10px;
+  }
+
+  .btn-card:hover {
+    opacity: 0.9;
+  }
+
+  .container {
+    display: grid;
+    margin: 0 auto;
+    gap: 50px 0;
+    width: 317px;
+  }
+
+  .project-1 {
+    display: flex;
+  }
+
+  .w-image {
+    width: 517px;
+    margin-right: 24px;
+  }
+
+  .right-block {
+    width: 380px;
+    margin-bottom: 40px;
+  }
+
+  h2 {
+    font-size: 32px;
+    font-family: 'Crete Round', serif;
+    margin: 0 0 24px 0;
+  }
+
+  .p-description {
+    font-size: 15px;
+    font-family: "inter", sans-serif;
+    font-weight: 400;
+    margin-top: 0;
+    margin-bottom: 12px;
+    line-height: 24px;
+  }
+
+  .languages {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    margin-bottom: 10px;
+    width: 236px;
+    background: white;
+    align-items: center;
+  }
```

Comment:
- [ ] Please ensure to align the languages to the left. they should all be left-aligned
- [ ] Kind make the same change for "Frameworks" list
- [ ] Kind make the same change for "Skills" list

## Your Design
![image](https://user-images.githubusercontent.com/18730106/145457328-40c42195-f3dc-4ef8-8700-873ea13268a5.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/145457456-e2f44333-5922-4d8c-b78c-758ede4505c5.png)

- [ ] Hi, Please also notice that your background color doesn't match correctly please address this issue